lib/plugins/cloud-watch.js

```diff
@@ -15,17 +15,22 @@ function register(skill, cloudwatch, eventMetric) {
     ],
     Namespace: 'StateMachine',
```

Comment:
let's provide more information on which parameters "should" be  overriden in the documentation. For example, the documentation provides an example where Namespace is overwritten, but there's no explanation on why that is. the rationale is that you want a separate namespace per skill